docs/security-features/network/open-ports.rst

```diff
@@ -1,11 +1,68 @@
 No Open Ports
+=============
+
+A port bound to a service that is actively listening for incoming packets on 
+a network is referred to as an "open port". Exposing open ports on untrusted 
+networks can pose a security risk if the listening service has a vulnerability 
+or is misconfigured.
+
+Historically, Ubuntu adhered to a "No Open Ports" policy, which 
+states that default installations of Ubuntu must have no listening network 
+services after initial install. Ubuntu continues to adhere as closely as possible 
+to this policy, with some exceptions. 
```

Comment:
It may be worthwhile stating when we started to adhere to this policy (i.e., the specific release in applicable).